diagram/yang-standalone/package.json

```diff
@@ -5,15 +5,17 @@
   "license": "Apache-2.0",
   "dependencies": {
     "reconnecting-websocket": "^3.0.7",
-    "sprotty": "0.2.1",
-    "yang-sprotty": "0.1.1",
+    "sprotty": "next",
```

Comment:
yang-lsp uses sprotty (server) pre-eclipse 0.3.0. The current `next` build of `eclipse/sprotty`(client) will not be compatible. If we don't want to update all yang tools, we should not allow to go higher than the last compatible version (0.6.0) or the specific next-version in the `yarn.lock` file